src/NuGet.Core/NuGet.Frameworks/NuGet.Frameworks.csproj

```diff
@@ -5,7 +5,7 @@
   <PropertyGroup>
     <Description>NuGet's understanding of target frameworks.</Description>
     <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
-    <TargetFrameworks Condition="'$(IsBuildOnlyXPLATProjects)' != 'true'">$(TargetFrameworks);net40</TargetFrameworks>
```

Comment:
Why do you need net45 here?

Reply:
not sure what you mean by "here".
if you mean on line 8 instead of 7...because of source build.
if you mean in nuget.frameworks at all... 
nuget.packaging.extraction -> nuget.configuration -> nuget.common -> nuget.frameworks

Reply:
The question is why do you need to add net45, because net40 should already be compatible.